doc/rest.md

```diff
@@ -0,0 +1,173 @@
+# REST
+
+When running and connected to a DotBot gateway, the `dotbot-controller`
+application provides a REST server to get information and control connected
+DotBots.
+
+The REST API is documented in the running `dotbot-controller` application itself
+at [http://localhost:8000/api](http://localhost:8000/api). This page also allows
+to play with the API directly from the browser.
+
+```{image} _static/images/pydotbot-ui-openapi.png
+:alt: Open API UI
+:class: bg-primary
+:width: 700px
+:align: center
+```
+
+## Prerequisites
+
+Make sure you already followed the [getting started](getting_started) page and
+have a functional setup with `dotbot-controller` running and connected to a
+nRF DK gateway.
+
+To interact with the REST API, you will use the Python
+[requests](https://pypi.org/project/requests/) package. You can install it on
+your computer using pip:
+
+```
+pip install -U requests
+```
+
+## The basics
+
+First let's start by fetching the information about available DotBots using
+the following script:
+
+```py
+import json
+import requests
+
+get_endpoint = "controller/dotbots"
+
+print(
+    json.dumps(
+        requests.get(
+            f"http://localhost:8000/{get_endpoint}"
+        ).json()
+    )
+)
+```
+
+If a DotBot is connected, this script should give an output similar to:
+```json
+[
+  {
+    "address": "9903ef26257feb31",
+    "application": 0,
+    "swarm": "0000",
+    "status": 2,
+    "mode": 0,
+    "last_seen": 1701244665.8099585,
+    "waypoints": [],
+    "waypoints_threshold": 40,
+    "position_history": []
+  }
+]
+```
+
+This is a list of all DotBots connected to the `dotbot-controller`. In the
+example above, there's only one DotBot connected.
+The `address` is an 8 bytes data that uniquely identifies a DotBot in the
+controller. The `status` tells if the DotBot is `Alive` (0), `Lost` (1) or
+`Dead` (2).
+
+If the DotBot `address` is already known, use the `contoller/dotbots/<address>`
```

Comment:
A bit ambiguous what a "known" address means. Known to whom?

Reply:
known to the controller

Reply:
I detailed a bit more